Filter.cs

```diff
@@ -109,8 +102,8 @@ static unsafe private Filter getFilter(void* data)
   public static unsafe sbyte* filter_get_name(void* data)
   {
     Module.Log("filter_get_name called", ObsLogLevel.Debug);
-    fixed (byte* logMessagePtr = Encoding.UTF8.GetBytes("C# Example Filter"))
```

Comment:
BTW, if this is not a constant string - it would not work because we return a pointer that is used **_after_** it is no longer fixed